index.html

```diff
@@ -524,11 +524,12 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <p data-dfn-for="PerformanceResourceTiming">On getting, the
 <dfn>domainLookupStart</dfn> attribute MUST return as follows:</p>
 <ol data-link-for="PerformanceResourceTiming">
-<li>The same value as <a>fetchStart</a>, if a <a href=
+<li>The same value as <a>fetchStart</a>, if no domain lookup was
+required to fetch the resources (e.g. due to the fact that a <a href=
 "https://tools.ietf.org/html/RFC7230#section-6.3">persistent
-connection</a> [[RFC7230]] is used or the resource is retrieved
-from <a data-cite="HTML#relevant-application-cache">relevant
-application caches</a> or local resources.</li>
+connection</a> [[RFC7230]] was used or in case the resource was
+retrieved from <a data-cite="HTML#relevant-application-cache">relevant
+application caches</a> or local resources).</li>
```

Comment:
do we also need this change the other places we mention "persistent connection"?

Reply:
Not all of them, but I did leave out domainLookupEnd...